cdap-docs/examples-manual/source/examples/spark-page-rank.rst

```diff
@@ -117,11 +124,21 @@ Querying the Results
 To query the *ranks* ObjectStore through the ``RanksService``, send a query via an HTTP
 request using the ``curl`` command. For example::
 
-  $ curl -w'\n' http://localhost:10000/v3/namespaces/default/apps/SparkPageRank/services/RanksService/methods/rank?url=http://example.com/page1
+  $ curl -w'\n' -X POST -d'{"url":"http://example.com/page1"}' http://localhost:10000/v3/namespaces/default/apps/SparkPageRank/services/RanksService/methods/rank
 
 You can also use the Command Line Interface::
 
-  $ cdap-cli.sh call service SparkPageRank.RanksService GET 'rank?url=http://example.com/page1'
+  $ cdap-cli.sh call service SparkPageRank.RanksService POST 'rank' body '{"url":"http://example.com/page1"}'
+
+Similarily to query *rankscount* ObjectStore through the ``TotalPagesPR``
+
+curl::
+
+  $ curl -w'\n' http://localhost:10000/v3/namespaces/default/apps/SparkPageRank/services/TotalPagesPR/methods/total/10
+
+Command Line Interface::
+
+  $ cdap-cli.sh call service SparkPageRank.TotalPagesPR GET 'total/10'
```

Comment:
Should we explain this syntax? (```total/10```)  Is this total number of pages with a page rank of ten?